src/util.h

```diff
@@ -141,3 +141,26 @@ static boost::filesystem::path findQmake() {
 
     return qmakePath;
 }
+
+static bool path_contains_file(boost::filesystem::path dir, boost::filesystem::path file) {
```

Comment:
We need this at additional spots. In a follow-up PR, we should replace that by calls to this function.

Naming convention violated (all other methods are `camelCase`).

Reply:
Fixed